src/octod/api/issues.d

```diff
@@ -141,7 +152,7 @@ Issue[] listIssues ( ref HTTPConnection connection, string repo )
     validateRepoString(repo);
 
     return connection
-        .get(format("/repos/%s/issues", repo))
+        .get(format("/repos/%s/issues?state=%s", repo, state))
```

Comment:
Sorry, should have said it more clearly from the start:

```D
import std.conv : asOriginalType;
`format("/repos/%s/issues?state=%s", repo, state.asOriginalType)`
```

(otherwise it will use enum name, not value)